openshift/templates/deploy.dc.yaml

```diff
@@ -45,11 +45,19 @@ parameters:
   - name: WPS_API_GLOBAL_SECRET
     description: Global secret
     value: wps-api-global
+  - name: USE_WFWX
+    description: Toggle use of WFWX API
+    value: "False"
```

Comment:
Shouldn't this be "True"?

Reply:
No. We have to keep using the json until we fully switch over to the WFWX. Two problems right now 1) there's no prod instance to point to 2) there may be a mismatch between stations on WFWX and station data we've generated.

We could just point everything in DEV to WFWX? I might cause some confusion when testing though, because of point 2 above.